src/js/components-react/DateTime.js

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import DateTimePicker, {registerLocale} from 'react-datepicker';
+import ja from 'date-fns/locale/ja';
+import zh from 'date-fns/locale/zh-CN';
+import en from 'date-fns/locale/en-US';
+
+class DateTime extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    ja.options.firstWeekContainsDate = 1;
+    ja.options.weekStartsOn = 0;
+    en.options.firstWeekContainsDate = 1;
+    en.options.weekStartsOn = 0;
+    zh.options.firstWeekContainsDate = 1;
+    zh.options.weekStartsOn = 0;
+    registerLocale('ja', ja);
+    registerLocale('zh', zh);
+    registerLocale('en', en);
+  }
+  _handleChangeDate = (date) => {
+    if (this.props.value === '') {
+      this.props.value = date;
+      this.setState({});
```

Comment:
Also here do you have any special reason to setState to empty object {}?

Reply:
Without setState, component not rerender when _handleChangeDate() ran . 
Do you have any idea to solve this problem ?

Reply:
It is the same as comment above. You just need to call this.props.onChange inside _handleChangeDate to pass the date to parent scope for handling. It could look something like this:
...
 _handleChangeDate = (date) => { 
  if(typeof this.props.onChange === 'function') {
    this.props.onChange(date);
  }
}

Thanks!